messagebird/client.py

```diff
@@ -296,5 +313,24 @@ def conversation_read_webhook(self, id):
         uri = CONVERSATION_WEB_HOOKS_PATH + '/' + str(id)
         return ConversationWebhook().load(self.request(uri, 'GET', None, CONVERSATION_TYPE))
 
+    def voice_recording_list_recordings(self, call_id, leg_id):
+        uri = VOICE_API_ROOT + '/' + VOICE_PATH + '/' + str(call_id) + '/' + VOICE_LEGS_PATH + '/' + str(leg_id) + '/' + VOICE_RECORDINGS_PATH
+        return VoiceRecordingsList().load(self.request(uri, 'GET'))
+
+    def voice_recording_view(self, call_id, leg_id, recording_id):
+        uri = VOICE_API_ROOT + '/' + VOICE_PATH + '/' + str(call_id) + '/' + VOICE_LEGS_PATH + '/' + str(leg_id) + '/' + VOICE_RECORDINGS_PATH + '/' + str(recording_id)
+        recording_response = self.request(uri, 'GET')
+        if recording_response['_links'] is not None:
```

Comment:
If this condition is **not** satisfied, what will be in `recording_response['data'][0]` ? Consider unit testing the case as well.